### PR TITLE
Added docs for the file system apis

### PR DESCRIPTION
**Pull request by diptanu**

### CODE REVIEW

File: website/source/docs/http/client-fs-cat.html.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+description: |-
+  The '/1/client/fs/cat` endpoint is used to read the contents of a file in an
+  allocation directory.
+---
+
+# /v1/client/fs/cat
+
+The `fs/cat` endpoint is used to read the contents of a file in an allocation
+directory. This API endpoint is hosted by the Nomad client and requests have to
+be made to the Nomad client where the particular allocation is running.
```
Comment (dadgar) [inline website/source/docs/http/client-fs-cat.html.md]: was placed. The task doesn't have to be running

File: website/source/docs/http/client-fs-cat.html.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  <dt>Parameters</dt>
+  <dd>
+    Path - The path relative to the root of the allocation directory. It
+    defaults to `/`
+  </dd>
+
+  <dt>Returns</dt>
+  <dd>
+    ```
+    2016/03/15 15:40:56 [DEBUG] sylog-server: launching syslog server on addr:
```
Comment (dadgar) [inline website/source/docs/http/client-fs-cat.html.md]: This is confusing as it makes it seem like that is what it is returning. Just say Returns: contents of requested file

File: website/source/docs/http/client-fs-cat.html.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  </dd>
+
+  <dt>Method</dt>
+  <dd>GET</dd>
+
+  <dt>URL</dt>
+  <dd>`/v1/client/fs/cat`</dd>
+
+  <dt>Parameters</dt>
+  <dd>
```
Comment (dadgar) [inline website/source/docs/http/client-fs-cat.html.md]: You missed the allocation as a parameter :)

Comment (diptanu) [inline website/source/docs/http/client-fs-cat.html.md]: Good catch!

**Pull request merged**